lib/app_config/app_config.dart

```diff
@@ -9,6 +9,9 @@ class AppConfig {
   int get maxCoinsEnabledAndroid => 50;
   int get maxCoinEnabledIOS => 20;
 
+  String get transactionWarningInfoUrl =>
+      "https://cryptonews.net/news/security/20792248/";
```

Comment:
might make sense to check if our marketing team has already release a blog post or ann around this and to refer to it. For the case that they didn't we might want to switch to a more know / well-established crypto media outlet.

Reply:
@ca333 
I've replaced it with the article link confirmed by the marketing team. It's ready to go live with the fix's release.

Reply:
https://github.com/KomodoPlatform/atomicdex-mobile/pull/23#issuecomment-1576573266